aepp/configs.py

```diff
@@ -147,6 +152,18 @@ def configure(
     config_object["private_key"] = private_key
     config_object["sandbox"] = sandbox
     header["x-sandbox-name"] = sandbox
+
+    # ensure we refer to the right environment endpoints
+    config_object["environment"] = environment
+    if environment == "prod":
+        endpoints["global"] = "https://platform.adobe.io"
```

Comment:
can we have these endpoints in a config just like we're doing with the notebooks

Reply:
These are available directly if you call the `aepp.config.endpoints` path.
So it seems that it is already available if needed.